notebooks/254-llm-chatbot/254-llm-chatbot.ipynb

```diff
@@ -510,7 +510,7 @@
     "            \"group_size\": 128,\n",
     "            \"ratio\": 0.8,\n",
     "        },\n",
-    "        \"chatglm2-6b\": {\n",
+    "        \"chatglm3-6b\": {\n",
```

Comment:
hi @eaidova could you also help to confirm chatglm3's quantization configurations ?

Reply:
it should be the same like for changpt2, @AlexKoff88  @ljaljushkin is it correct?

Reply:
@eaidova we didn't search the optimal quantization config for chatglm3. Probably, it should be as chatglm2, since the default ratio r=1.0 might be too high.

Reply:
chatglm3 is fine-tuned with more data version of chatglm2, that is why I say that here should not be changes